client/rrsim_test.cpp

```diff
@@ -287,7 +286,9 @@ bool CLIENT_STATE::rr_simulation() {
         int last_active_size = active.size();
         int last_proj_active_size = pbest->active.size();
 
-        // remove *rpbest from active set,
+        {
```

Comment:
Extra scopes are not necessary here.
Either reuse `it` variable or refactor this code to move these two cycles into separate functions.
Also please adjust aligning.

Reply:
If you do not see anything you want to change in the current BOINC tree then please this PR.

Reply:
"close" was missing.

Reply:
My comment is still valid